libnavigation-util/src/main/java/com/mapbox/navigation/utils/internal/ConnectivityHandler.kt

```diff
@@ -0,0 +1,23 @@
+package com.mapbox.navigation.utils.internal
+
+import android.util.Log
+import com.mapbox.common.NetworkStatus
+import com.mapbox.common.ReachabilityChanged
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.channels.ReceiveChannel
+
+class ConnectivityHandler : ReachabilityChanged {
+    private val networkStatusChannel = Channel<Boolean>(Channel.CONFLATED)
+
+    fun getNetworkStatusChannel(): ReceiveChannel<Boolean> = networkStatusChannel
+
+    override fun run(status: NetworkStatus) {
+        Log.e("DEBUG", "DEBUG ConnectivityHandler.status=$status")
```

Comment:
Use `Logger`.

Reply:
This was only for debugging purposes. I wasn't planning to leave it out afterwards - don't want to spam out the `Logcat`, can do if you think otherwise.

Reply:
Ah, gotcha. I actually do think that it would be beneficial to log the connectivity state.